src/runtime/auto-prompt-manager.ts

```diff
@@ -690,12 +699,15 @@ export class AutoPromptManager {
       )
     );
 
+    if (actions.length === 0) {
+      return;
+    }
+
     // FrequencyCapConfig may be undefined ONLY for subscription openacess
     // content. If so, display first valid action.
-    if (
-      this.isSubscription_(autoPromptType) ||
-      !this.isValidFrequencyCap_(frequencyCapConfig)
-    ) {
+    const isLockedContent =
+      this.isSubscription_(autoPromptType) && this.pageConfig_.isLocked();
```

Comment:
Would `isClosable` be more accurate than subscription && locked?
What's the expected behavior for non-subscription && (isClosable == false)?

Reply:
Discussed offline, extending to `isClosable` prompts.